packages/graph/workbooks/session.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQueryable, graphPost } from "../graphqueryable.js";
+import { InjectHeaders, body } from "@pnp/queryable";
+import { IWorkbookWithSession, WorkbookWithSession } from "./types.js";
+import { _DriveItem } from "../files/types.js";
+import {
+    WorkbookSessionInfo
+} from "@microsoft/microsoft-graph-types";
+
+export async function getWorkbookSession(this: _DriveItem, persistChanges: boolean): Promise<IWorkbookWithSession> {
```

Comment:
If the only place this will really get used is in the property in the other file you can code it there directly. i.e. remove the export here and put the code there. But if this is a function that will be used on its own this makes sense.